tools/cellranger_mkref.cwl

```diff
@@ -0,0 +1,60 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: cellranger_mkref
+doc: "Make a custom reference for cellranger"
```

Comment:
Might want to make it clear what this tool does functionally. Explain how providing the `attribute` field changes the behavior of this tool.